playbook-website/app/views/pages/kit_show.html.erb

```diff
@@ -38,7 +38,7 @@
       <% if pb_doc_has_kit_type?(@kit, "react") %>
         <%= pb_rails("nav/item", props: { text: "React", link: kit_show_reacts_path(@kit), active: action_name == 'kit_show_react' }) %>
       <% end %>
-      <% if Rails.env.development? %>
```

Comment:
I will add these back in after testing has been approved. Because review envs are "Production" this could not be tested with the necessary `Rails.env.development?` checks.

In Nitro we use feature toggles to get around this, but its simple enough to add these back in after

Reply:
these checks have been added back, but I left it so you can force the url to see the swift examples